source/includes/_api.md

```diff
@@ -51,11 +53,39 @@ It provides developers a method to enable or disable certain features or functio
 `applicationVersion` | No | String of maximum length **30 characters**.
 `disablePrecalltest` | No | by default the value is `false`.
 `siteID` | No | String (256 bytes).
+`collectLegacyStats` | No | by default the value is `true`.
+`applicationIDs` | No | JSON object.
 
 <aside class="error">
 Setting `disableBeforeUnloadHandler` to `true` disengages callstats.js's `window.onbeforeunload` handler, and you will need to send the fabricTerminated event for each active PeerConnection. See more details on `fabricTerminated` [event](#step-5-optional-sendfabricevent)
 </aside>
 
+### JSON for applicationIDs
+
+```javascript
+var applicationIDs = {
+  customerID: "customer ID",
+  pbxID: "PBX ID",
+  pbxExtensionID: "PBX Extension ID",
+  fqExtensionID: "fully qualified extension ID",
+  meetingsName: "Meeting Name",
+  tenantID: "Tenant ID",
+  productName: "Product Name",
+  serverName: "Server Name",
+};
```

Comment:
is this done before initialize? could you add that part to this example?

Reply:
It is shown in the intitialize section, with example, the javascript blob will be in right side.

Reply:
<img width="1515" alt="Screenshot 2020-11-25 at 15 02 08" src="https://user-images.githubusercontent.com/12692403/100231115-4014b780-2f2f-11eb-84e3-65036b063b88.png">